mkdocs.yml

```diff
@@ -125,6 +126,8 @@ nav:
       - Jenkins: ./reference/cicd/jenkins.md
     - SSL:
       - Kubernetes: ./reference/ssl/kubernetes-with-cert-manager.md
+    - Logs:
```

Comment:
I think it would still work if this was collapsed down into one page instead of an empty top-level.